src/js/Toolbar/L.PM.Toolbar.js

```diff
@@ -563,7 +563,16 @@ const Toolbar = L.Class.extend({
     this.changeControlOrder();
     return control;
   },
-
+  controlExists(name) {
+    return Boolean(this.getButtons()[name]);
+  },
```

Comment:
Would it be ok for you to implementend `getButton(name)`? It returns `undefined` if it doesn't exists.